packages/datagrid/src/datagrid.ts

```diff
@@ -6721,6 +6762,31 @@ export namespace DataGrid {
     headers: 'none',
     warningThreshold: 1e6
   };
+
+  /**
+   *
+   */
+  export interface IBodyRegion {
+    /**
+     * First row of the region
+     */
+    r1: number;
```

Comment:
Thanks for you suggestion! Reading a bit more the rest of the API it seems `row`, `rowspan`, `column`, `columnspan` would be more homogeneous with the rest? What do you think?

Reply:
`rowspan` seems to be used for cells which span multiple rows. I would not use it here.

QT uses `topRow`, `bottomRow` etc https://doc.qt.io/qt-6/qml-qtquick-tableview.html#topRow-prop but these are not in an object so that's why they need "row" suffix.

Some other reference points:
- GTK does not seem to have an API for this in `Table` nor `Grid` components
- Windows .Net DataGrid only exposes the first visible cell: [`firstdisplayedcell`](https://learn.microsoft.com/en-us/dotnet/api/system.windows.forms.datagridview.firstdisplayedcell).
- MUI seems to only have [`getVisibleColumns()`](https://mui.com/x/api/data-grid/grid-api/#grid-api-prop-getVisibleColumns)

QT, GTK and MUI uses `rowSpan`/`colSpan` for span of cells, so mixing these concepts here would confuse developers coming from other frameworks.

Reply:
Sounds good! Thanks for the references.

Let's go for `topRow`, `bottomRow` and `topColumn`, `bottomColumn` then :) 

FYI, this new API would make https://github.com/bloomberg/ipydatagrid/pull/498 possible without touching private APIs.